tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,16 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
Good! You could also test for 
```python
    assert len(vendor.inventory) == 3
```
just to make sure the list wasn't changed unexpectedly.